doc/lua_api.txt

```diff
@@ -2380,6 +2380,10 @@ Strings that need to be translated can contain several escapes, preceded by `@`.
       * nil: return all entries,
       * true: return only subdirectory names, or
       * false: return only file names.
+* `minetest.safe_file_write(path, content)`: returns boolean indicating success
+    * Replaces contents of file at path with new contents in a safe (atomic) way.
+      Use this instead of below code when writing e.g. database files:
+      local f = io.open(..., "w"); f:write(newtext); f:close()
```

Comment:
Code example, should be using the inline code notation:
\`local f = io.open(..., "w"); f:write(newtext); f:close()`